indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -215,8 +219,8 @@
                 class: 'title',
                 text: category.title
             }));
-            if (withBreadcrumbs && category.path.length > 0) {
-                $categoryTitle.append(self._buildBreadcrumbs(category.path, clickableBreadcrumbs));
+            if (withBreadcrumbs && category.parent_path.length > 0) {
```

Comment:
`category.parent_path.length > 0` since you are using `category.parent_path.length` below (to be more consistent)